.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+**This issue tracker is a tool to address bugs in Flask-SQLAlchemy itself.
+Please use the [#pocoo IRC channel on freenode](http://flask.pocoo.org/community/irc/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/flask-sqlalchemy?sort=linked) for general
+questions about using Flask-SQLAlchemy or issues not related to Flask-SQLAlchemy.**
+
+If you'd like to report a bug in Flask-SQLAlchemy, fill out the template below.
+Provide any extra information that may be useful / related to your problem.
+Ideally, create a [minimal, complete, and verifiable example (MCVE)](https://stackoverflow.com/help/mcve), which helps us
+understand the problem and helps check that it is not caused by something in
+your code.
+
+Flask-SQLAlchemy is a thin wrapper that combines [Flask](https://github.com/pallets/flask) and [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy). Please check
+if your issue has already been reported as a [SQLAlchemy issue on Github](https://github.com/sqlalchemy/sqlalchemy/issues).
```

Comment:
I'd leave this out:

> Please check
if your issue has already been reported as a [SQLAlchemy issue on Github](https://github.com/sqlalchemy/sqlalchemy/issues).

And just say: "Please make sure your issue is actually with Flask-SQLAlchemy and not Flask or SQLAlchemy before submitting."

Reply:
@rsyring I hear what you're saying, but what's the action item for beginners?

Reply:
Sorry, I'm not exactly sure what you are asking.  For a true beginner, IMO, they are likely to just make the issue post anyway b/c they don't really understand the layers or libraries involved.  But for an intermediate or advanced user it serves as a reminder, hopefully, to double-check what the error is and try to make sure it's an FSA issue before submitting.

Maybe, it's to suggest they google for their issue with just "SQLAlchemy" and/or point them at the SQLAlchemy docs.  My gut is, may problems people encounter with FSA end up really being SQLAlchemy usage errors.

Reply:
Maybe "Please make sure your issue is actually with Flask-SQLAlchemy and not Flask or SQLAlchemy before submitting. Consider checking the traceback to see if the error is coming from SQLAlchemy, or if your issue has already been reported as a [SQLAlchemy issue on Github](https://github.com/sqlalchemy/sqlalchemy/issues)"

Reply:
:+1: